client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -109,6 +109,9 @@ module.exports = React.createClass( {
 
 	render: function() {
 		var name = this.getProductName();
+		if ( this.props.cartItem.bill_period !== -1 ) {
```

Comment:
`this.props.cart.hasPendingServerUpdates` almost would have worked, but it caused all cart items to lose the "- annual period" suffix, even when those cart items are loaded. I updated the commit to use the presence check.

:+1: